airbyte-integrations/connectors/source-s3/v4_main.py

```diff
@@ -27,17 +28,21 @@ def get_source(args: List[str]):
         )
     except Exception:
         print(
-            AirbyteMessage(
-                type=Type.TRACE,
-                trace=AirbyteTraceMessage(
-                    type=TraceType.ERROR,
-                    emitted_at=int(datetime.now().timestamp() * 1000),
-                    error=AirbyteErrorTraceMessage(
-                        message="Error starting the sync. This could be due to an invalid configuration or catalog. Please contact Support for assistance.",
-                        stack_trace=traceback.format_exc(),
-                    ),
-                ),
-            ).json()
+            orjson.dumps(
+                AirbyteMessageSerializer.dump(
```

Comment:
I'm not sure I follow the nested "dump" statements here. 🤔

Reply:
Serializer `dump` (transform) `dataclass` -> `dict`
orjson `dumps` `dict` -> JSON

Reply:
Got it. That makes sense. I think I misread the serializer 'dump' as dumping to json string.